http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -435,11 +468,16 @@ class DataprocInterpreter[F[_]: Timer: Async: Parallel: ContextShift: Logger](
 
     // Don't delete the staging bucket so the user can see error logs.
 
-    val deleteCluster =
-      Async[F].liftIO(IO.fromFuture(IO(gdDAO.deleteCluster(googleProject, clusterName)))).attempt.flatMap {
-        case Left(e) => Logger[F].error(e)(s"Failed to delete cluster ${googleProject.value} / ${clusterName.asString}")
-        case _       => Logger[F].info(s"Successfully deleted cluster ${googleProject.value} / ${clusterName.asString}")
-      }
+    val deleteCluster = {
+      googleDataprocService
+        .deleteCluster(googleProject, config.dataprocConfig.regionName, DataprocClusterName(clusterName.asString))
+        .attempt
+        .flatMap {
+          case Left(e) =>
+            Logger[F].error(e)(s"Failed to delete cluster ${googleProject.value} / ${clusterName.asString}")
```

Comment:
is there any traceId we can put in mdc?

Reply:
Yeah I think so.. can switch to `StructuredLogger` as well